scipy/stats/stats.py

```diff
@@ -3999,6 +3996,51 @@ def pearsonr(x, y):
     >>> stats.pearsonr([1, 2, 3, 4, 5], [10, 9, 2.5, 6, 4])
     (-0.7426106572325057, 0.1505558088534455)
 
+    It is important to keep in mind that no correlation does not imply
+    independence unless (x, y) is Gaussian. Correlation can even be zero
+    when there is a very simple dependence structure: if x follows a
+    standard normal distribution, let y = abs(x). A simple check
+    confirms that the correlation is close to zero:
+
+    >>> x = stats.norm.rvs(size=500)
+    >>> y = np.abs(x)
+    >>> stats.pearsonr(x, y)
+    (-0.016172891856853524, 0.7182823678751942) # may vary
+
+    Indeed, since the expectation of x is zero, cov(x, y) = E[x*y]. This equals
+    E[x*abs(x)] which is zero by symmetry.
+
+    If the relationship between x and y is non-linear, the correlation
+    coefficient can be misleading. For example, if x has a standard normal
```

Comment:
```suggestion
    A nonzero correlation coefficient can also be misleading. 
    For example, if x has a standard normal
```
The previous example is just another example of the correlation coefficient being misleading when there is a nonlinear relationship between x and y . I think we want to highlight the difference: in that case the correlation coefficient was zero even when there was a simple relationship, whereas here there is a very nonzero correlation even though the two are not really strongly correlated.